logfilter/logfilter.go

```diff
@@ -51,11 +50,10 @@ var (
 	}
 )
 
-var environmentTag *string
+var environmentTag *string = new(string)
 
 func main() {
-	environmentTag = flag.String("environment", "", "set the environment tag to use in the outputted json")
-	flag.Parse()
+	*environmentTag = os.Getenv("ENV")
```

Comment:
I agree, but in this case I think it's not needed because os.Getenv("ENV") will return "" if the environment variable doesn't exist/is not set, and theenvironmentTag variable is checked for empty, below at line 101:
`	
if *environmentTag != "" {
		m["environment"] = *environmentTag
	}
`